controllers/drplacementcontrol_controller.go

```diff
@@ -888,31 +888,47 @@ func (r *DRPlacementControlReconciler) getVRGsFromManagedClusters(drpc *rmn.DRPl
 	annotations[DRPCNameAnnotation] = drpc.Name
 	annotations[DRPCNamespaceAnnotation] = drpc.Namespace
 
+	var failedClusterToQuery string
+
+	var clustersQueriedSuccessfully int
+
 	for _, drCluster := range rmnutil.DrpolicyClusterNames(drPolicy) {
 		vrg, err := r.MCVGetter.GetVRGFromManagedCluster(drpc.Name, drpc.Namespace, drCluster, annotations)
 		if err != nil {
 			// Only NotFound error is accepted
 			if errors.IsNotFound(err) {
 				log.Info(fmt.Sprintf("VRG not found on %q", drCluster))
+				clustersQueriedSuccessfully++
 
 				continue
 			}
 
-			if drpc.Spec.Action == rmn.ActionFailover && drpc.Spec.FailoverCluster != drCluster {
```

Comment:
Previously initial deployment would fail in this function if even one cluster is down. Now,
	- If the cluster that is down is not the failoverCluster value (which maybe an empty string) it would continue

Not sure if that is an issue though.

Reply:
Yeah, let it continue.  It will be trapped in the RunInitialDeployement.